### ISSUE TITLE
Contact pgp key has expired

### ISSUE THREAD

**Issue by tobias** [clojars/clojars-web#657]
>  Hi! I'm Paul and this email was sent by a friendly robot called Expirybot.
> 
> This is a reminder that your PGP key expires on Sunday 03 September 2017.
> 
> fingerprint: AF97 A8AF 8C77 F047 8339  C6C9 6F4A 6C02 3D52 DD95
> long key id: 0x6F4A6C023D52DD95
> 
> Here's the key listed in the MIT public keyserver:
> 
> https://pgp.mit.edu/pks/lookup?op=vindex&search=0x6F4A6C023D52DD95
> 
> == Extend your key expiry date ==
> 
> You can extend your key's expiry date and push it to the keyservers.
> 
> You don't need to create a new key.
> 
> I've written a few short guides with screenshots for different software:
> 
> - Enigmail with Thunderbird: https://www.paulfurley.com/extend-pgp-key-expiry-with-enigmail-thunderbird
> - GPG Suite (GPG Keychain): https://www.paulfurley.com/extend-pgp-key-expiry-with-gpg-suite-gpg-keychain
> - GNU Privacy Guard (GnuPG): https://www.paulfurley.com/extend-pgp-key-expiry-with-gpg
> 
> (I'm currently working on Mailvelope - any others?)
> 
> == Remember to push your updated key to the keyservers ==
> 
> Once you've extended the expiry date, don't forget to push your key to the keyservers again so everyone can see the update.
> 
> I hope this email has been helpful. If this feels like spam, I'm sorry :(
> 
> Kind regards,
> 
> Paul
> 
> --
> Paul M Furley => Engineer / Developer
> 
> Let's make it better! Trying to improve the world by making :)
> 
> - Website: https://www.paulfurley.com/expirybot-emails-pgp-users-before-their-key-expires/
> - Twitter: https://twitter.com/paul_furley
> - PGP: A999 B749 8D1A 8DC4 73E5  3C92 309F 635D AD1B 5517

**Comment by technomancy** [comment]
I can generate a new keypair if we can agree on a means whereby the new key should be distributed. Is encrypting the new key with the old key and mailing it to the folks on the security team a decent way to do it? I'm not sure who all the people on the security team are.

**Comment by tobias** [comment]
I generated a new keypair today, updated https://clojars.org/security to link to the public key (https://github.com/clojars/clojars-web/commit/6158efd2c2c5d9b403b9f3eec3bd78a80bd8b2b7),
and pushed it to pgp.mit.edu. Currently the security team is only @danielcompton and I, so I just shared the private key with him. If you're interested in getting back in to working on Clojars again, I'm happy to share the key with you as well.
